Frontend/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "bootstrap",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "apollo-boost": "^0.1.12",
+    "apollo-cache-inmemory": "^1.3.9",
+    "cypress": "^3.1.3",
+    "enzyme": "^3.8.0",
```

Comment:
enzyme and cypress stuff should go under `devDependencies` instead since they are not used in the production build 🏗